nmdc_automation/workflow_automation/activities.py

```diff
@@ -3,6 +3,13 @@
 from .workflows import Workflow
 from semver.version import Version
 
+# TODO: Berkley refactoring:
+#   The load_activities method will need to be modified to handle DataGeneration objects
```

Comment:
DataGeneration is an abstract class, we should to its subclasses NucleotideSequencing or MassSpectrometry